app/src/lib/emails/AdminNotificationTemplate.tsx

```diff
@@ -0,0 +1,185 @@
+import { ExcelFileIcon } from "@/components/icons";
+import { CityAttributes } from "@/models/City";
+import { UserAttributes } from "@/models/User";
+import { UserFileAttributes } from "@/models/UserFile";
+import { bytesToMB } from "@/util/helpers";
+import { UserFileResponse } from "@/util/types";
+import {
+  Body,
+  Button,
+  Container,
+  Head,
+  Html,
+  Link,
+  Preview,
+  Text,
+  Img,
+  Font,
+  Hr,
+} from "@react-email/components";
+import { FaFileExcel } from "react-icons/fa";
+
+export default function AdminNotificationTemplate({
+  user,
+  files,
+  adminNames,
+}: {
+  user: { name: string; email: string; cityName: string };
+  files: UserFileResponse[];
+  adminNames: string;
+}) {
+  const host = process.env.HOST ?? "http://localhost:3000";
```

Comment:
Can we extract a Config script/ singleton that handles env vars and setting default values for them? This is littered throughout the codebase...

Reply:
Doesn't have to be in this PR but let's do it soon.